### ISSUE TITLE
Xcode 8.0 playground can not show the cursor when coding

### ISSUE THREAD

**Issue by duanxu520** [XVimProject/XVim#998]
# Description
Xcode 8.0 playground can not show the cursor when coding

# Environments
  - Xcode Version [  8A218a  ]
  - XVim branch and revision [  master ]
  - Input source language [  swift  ]

**Comment by adriandeveloper** [comment]
Confirmed.

**Comment by jack2gs** [comment]
I got the same problem. And some intelligent prompt come out slowly than before, for example, when you type "var". BTW, it's swift.

**Comment by ambujpunn** [comment]
Is there a fix for this?!

**Comment by shengithub** [comment]
Still no fix on this issue?

**Comment by ambujpunn** [comment]
Nope it's been fixed! Make sure to pull from the latest

**Comment by rudedogg** [comment]
@ambujpunn Are you sure? I just pulled from master, installed, and still can't see the cursor in normal mode :(.

**Comment by ambujpunn** [comment]
@rudedogg what version of xcode are you on? did you make sure to ```make uninstall```, close xcode, ```make```, open xcode and load bundle?

**Comment by rudedogg** [comment]
@ambujpunn No I hadn't run `make uninstall`. I just went through it that way though and still no luck :(.

**Comment by jack2gs** [comment]
@ambujpunn I just did as what you mentioned. But it still didn't work. BTW, it's Xcode 8.1.

**Comment by ambujpunn** [comment]
@jack2gs @rudedogg have you created a new code sign certificate? You'll have to recodesign xcode in order for xvim to work:

https://github.com/XVimProject/XVim/blob/master/INSTALL_Xcode8.md

**Comment by jack2gs** [comment]
@ambujpunn Nope. Everything seems fine, so I didn't resign Xcode.  But I will try it again. And later I'll tell you the result. Thanks!

**Comment by jack2gs** [comment]
@ambujpunn I'v resigned the Xcode and reinstalled XVim. But still not working.

**Comment by shengithub** [comment]
Same thing here, resigned Xcode 8.1 and reinstalled Xvim and still not working

**Comment by yllziv** [comment]
mark, I already resigned Xcode 8.1 and so so...., but not work

**Comment by rudedogg** [comment]
https://github.com/XVimProject/XVim/pull/1016

**Comment by tocology** [comment]
Same thing here, I'm using Xcode 8.2.1 that looks latest version.

**Comment by ChrisBuchholz** [comment]
Still doesn't work for me either, neither with xcode 8.2 and xcode 8.2.1 with latest xvim from master.

**Comment by flatcoke** [comment]
I can't still work it out when I use Xcode 8.2.1

**Comment by tidbeck** [comment]
Have you built XVim from the develop branch as the PR in #1016 isn't merged to master yet?

**Comment by mohoromitch** [comment]
The develop branch is working well for me.
